training/model_evaluation/src/evaluate_model.py

```diff
@@ -251,7 +266,8 @@ def test_model():
             model_uri=model_uri,
             config_file=args.config_file_name,
             metrics_config=met_config,
-            device=args.device
+            device=args.device,
+            batch_size=args.batch_size
```

Comment:
how u r expecting predict & metrics computation in batches in Component?
Will the aggregation logic for computation be picked from azureml-metrics custom vision metrics batch implementation?